Dockerfile

```diff
@@ -1,24 +1,25 @@
 # To use or update to a ruby version, change {BASE_RUBY_IMAGE}
-ARG BASE_RUBY_IMAGE=ruby:3.2.2-alpine3.16
+ARG BASE_RUBY_IMAGE=ruby:3.2.2-alpine
```

Comment:
Can it be pinned to 3.19? https://hub.docker.com/layers/library/ruby/3.2.2-alpine3.19/images/sha256-f94a95d5b837ca713c2b513a4a7125e773a708a4db21156b2a95a367efc53ead?context=explore